src/static/makefile

```
@@ -9,7 +9,7 @@ define compile_bin
 endef
 
 define compile_objs
-	gcc $(CFLAGS) -fPIC -c -o "$@" $< $(IDIRS:%=-I%)
+	gcc $(CFLAGS) -fPIC -c -o "$@" $< $(IDIRS:%=-I%) $(LIBS:%=-l%)
```

Este de acá no hace falta, porque se linkean al compilar a código máquina con el otro `LIBS` 👍🏻 

```suggestion
	gcc $(CFLAGS) -fPIC -c -o "$@" $< $(IDIRS:%=-I%)
```